central/image/service/service_impl.go

```diff
@@ -691,18 +915,170 @@ func (s *serviceImpl) enrichWithVulnerabilities(img *storage.Image, request *v1.
 	return err
 }
 
+func (s *serviceImpl) enrichLocalImageV2Internal(ctx context.Context, request *v1.EnrichLocalImageInternalRequest) (*v1.ScanImageInternalResponse, error) {
```

Comment:
Given that we're not coordinating w/ Sensor, and a V1 image is expected in return - the response image SHOULD contain all known Names for the image - otherwise cache misses will occur sensor side - which has caused outages for users in the past.

Reply:
I see, I wasn't aware of the scale and assumed those cache misses to be a mild annoyance. In this case, I will update the enricher to send add all known image names too.